data/language/en-GB.txt

```diff
@@ -4471,6 +4471,8 @@ STR_6161    :Gridlines display toggle
 STR_6162    :Spinning Wild Mouse
 STR_6163    :Mouse shaped cars speed through tight corners and short drops, gently spinning around to disorientate the riders
 STR_6164    :{WHITE}{CROSS}
+STR_6165    :Use Vsync
+STR_6166    :{SMALLFONT}{BLACK}Synchronizes each frame displayed to the monitor's refresh rate, preventing screen tearing.
```

Comment:
Synchroni**s**es

Reply:
TIL. I'll fix this.

Reply:
"Synchronizes" is correct in American English, but you're putting text into a file in _British English_.

Reply:
I figured that's what happened. I remember now Z -> S being a problem before. If I make a mistake like that, it's likely because I forgot that there was even a difference.